src/bin/add/args.rs

```diff
@@ -96,7 +98,8 @@ impl Args {
                              } else if let Some(ref path) = self.flag_path {
                                  dependency.set_path(path)
                              } else {
-                                 let v = try!(get_latest_version(&self.arg_crate));
+                                 let mut v = try!(get_latest_version(&self.arg_crate));
+                                 v = self.get_upgrade_prefix().to_owned() + &v;
```

Comment:
`v` is only used in this scope, right? It might look nicer to use `format!()` then:

```rust
let v = format!(
    "{prefix}{version}",
    prefix = self.get_upgrade_prefix().unwrap_or("",) // assumes this returns an Option as commented below
    version = try!(get_latest_version(&self.arg_crate))
)
```